lldb/include/lldb/Core/Progress.h

```diff
@@ -21,6 +21,12 @@
 
 namespace lldb_private {
 
+/// Enum to indicate the origin of a progress event, internal or external.
+enum class ProgressOrigin : uint8_t {
+  eLLDBInternal = 0,
+  eExternal = 1,
```

Comment:
Let's go with `eInternal` and `eExternal`. We generally avoid putting `LLDB` in names and the whole enum is part of `lldb_private`. I would also suggest to move it into the progress class and drop the `Progress` prefix so we have `Progress::Origin:eInternal` and `Progress::Origin:eExternal`.